src/pages/instances/Events.tsx

```diff
@@ -47,6 +51,7 @@ const Events: FC = () => {
         void queryClient.invalidateQueries({
           queryKey: [queryKeys.operations, event.project],
         });
+        void refetchOperations();
```

Comment:
I wonder if we should debounce the refetch. Especially with bulk actions there can be very many updates in short time.